<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="5" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="20878" started-at="2020-10-09T05:04:56Z" finished-at="2020-10-09T05:05:17Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="20878" started-at="2020-10-09T05:04:56Z" finished-at="2020-10-09T05:05:17Z">
      <class name="com.ijmeet.functionaltests.HomePageTests">
        <test-method status="PASS" signature="openBrowser()[pri:0, instance:com.ijmeet.functionaltests.HomePageTests@7193666c]" name="openBrowser" is-config="true" duration-ms="17818" started-at="2020-10-09T05:04:56Z" finished-at="2020-10-09T05:05:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method status="PASS" signature="verifyIsHomePageVisible()[pri:0, instance:com.ijmeet.functionaltests.HomePageTests@7193666c]" name="verifyIsHomePageVisible" duration-ms="56" started-at="2020-10-09T05:05:14Z" finished-at="2020-10-09T05:05:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyIsHomePageVisible -->
        <test-method status="PASS" signature="verifyIsJoinMeetingPageAvailable()[pri:0, instance:com.ijmeet.functionaltests.HomePageTests@7193666c]" name="verifyIsJoinMeetingPageAvailable" duration-ms="2589" started-at="2020-10-09T05:05:14Z" finished-at="2020-10-09T05:05:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyIsJoinMeetingPageAvailable -->
        <test-method status="FAIL" signature="verifyIsLogoPresent()[pri:0, instance:com.ijmeet.functionaltests.HomePageTests@7193666c]" name="verifyIsLogoPresent" duration-ms="185" started-at="2020-10-09T05:05:16Z" finished-at="2020-10-09T05:05:16Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.ijmeet.functionaltests.HomePageTests.verifyIsLogoPresent(HomePageTests.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyIsLogoPresent -->
        <test-method status="FAIL" signature="verifyPresenceOfAllMenuItems()[pri:0, instance:com.ijmeet.functionaltests.HomePageTests@7193666c]" name="verifyPresenceOfAllMenuItems" duration-ms="9" started-at="2020-10-09T05:05:16Z" finished-at="2020-10-09T05:05:16Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.ijmeet.util.WaitFor.minTime(WaitFor.java:15)
at com.ijmeet.functionaltests.HomePageTests.verifyPresenceOfAllMenuItems(HomePageTests.java:58)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyPresenceOfAllMenuItems -->
        <test-method status="PASS" signature="verifyTitleOnHomePage()[pri:0, instance:com.ijmeet.functionaltests.HomePageTests@7193666c]" name="verifyTitleOnHomePage" duration-ms="31" started-at="2020-10-09T05:05:16Z" finished-at="2020-10-09T05:05:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyTitleOnHomePage -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.ijmeet.functionaltests.HomePageTests@7193666c]" name="closeBrowser" is-config="true" duration-ms="124" started-at="2020-10-09T05:05:16Z" finished-at="2020-10-09T05:05:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.ijmeet.functionaltests.HomePageTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
